test/distributions/sungeun/dataPar2.prediff

```diff
@@ -2,7 +2,7 @@
 # sort the output to handle the numa case where iterator debug output is in a
 # non-deterministic order. Don't sort for non-numa 
```

Comment:
I'm about to push a change that fixes these comments in prediff and skipif files.  Pushing the change will eventually hide this github comment.